sycl/test/warnings/conversion_deprecation.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clangxx %fsycl-host-only -fsyntax-only -sycl-std=2020 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
+// RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
+// RUN: %clangxx %fsycl-host-only -fsyntax-only -sycl-std=2017 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
```

Comment:
Could you please clarify if we really need to check this warning in all modes? Suggest checking in default mode only.

Reply:
I took these lines from an example of another test, sorry for the mistake, I'll leave only the default mod:
```
// RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
```

Reply:
Drop `-o %t.out`. It's meaningless with `-fsyntax-only`.

Reply:
> Drop `-o %t.out`. It's meaningless with `-fsyntax-only`.

Done, thanks for the note